raiden/api/rest.py

```diff
@@ -176,13 +197,23 @@ def get_tokens_list(self):
         result = self.tokens_list_schema.dumps(tokens_list)
         return result
 
-    def get_new_events(self):
-        raiden_service_result = self.get_new_events()
-        assert isinstance(raiden_service_result, list)
+    def get_network_events(self, from_block, to_block):
+        raiden_service_result = self.raiden_api.get_token_added_events(
```

Comment:
Why doesn't the `raiden_api` class use the same method name `get_network_events` vs. `get_token_added_events`?

As I understand this endpoint will return all the events that were emitted by the network registry, so future events could be added.